### PR TITLE
update ui-index and ui-dashboard-index to use 2.10 UI builds

### PR DESCRIPTION
**Pull request by mantis-toboggan-md**
## Problem
Currently Rancher head builds are loading ui for the next 2.9 release. 
 
## Solution
Similar to https://github.com/rancher/rancher/pull/46309, ui-index and ui-dashboard-index settings need to be updated to point back to ui built from the rancher/dashboard `master` branch, which tracks 2.10
 
### Manual Testing
Updated the settings in an existing rancher instance with the new values and ensured both UIs load correctly

### CODE REVIEW

**Comment by richard-cox** [comment]
We should pause on this to give the vue3 changes time to settle in

**Comment by gaktive** [comment]
QA is asking to get this in to help with their Jenkins automation testing for Vue3. At this stage, can we consider merging this to try pointing to what will be 2.10?

**Review by richard-cox — APPROVED** [review]
The updates are correct, so giving a tick. Just needs consensus from the vue3 stars @codyrancher , @rak-phillip and @cnotv

**Comment by cnotv** [comment]
> The updates are correct, so giving a tick. Just needs consensus from the vue3 stars @codyrancher , @rak-phillip and @cnotv

I would prefer to go full green again https://github.com/rancher/dashboard/issues/12051
If this eases QA and other checks, then we can prioritize restoring the head.

**Comment by rak-phillip** [comment]
My vote is to merge. I think that the Vue3 branch has reached an acceptable level to stability to make the switch.

**Review by rak-phillip — APPROVED** [review]

**Comment by rak-phillip** [comment]
We're holding off this change until rancher/dashboard#12071 is merged.

**Pull request merged**
